saleor/checkout/models.py

```diff
@@ -138,7 +138,7 @@ class Checkout(ModelWithMetadata):
         max_digits=5, decimal_places=4, default=Decimal("0.0")
     )
 
-    price_expiration = models.DateTimeField(default=timezone.now)
+    price_expiration = models.DateTimeField(null=True, default=None, blank=True)
```

Comment:
What is a reason for giving a possibility to set this to None? Would be good to describe this change at least in PR's describtion

Reply:
It was much easier to deal with it in tests, but eventually I refactored tests so that timezone.now is used for invalidation